pallets/types/src/substrate.rs

```diff
@@ -97,7 +97,12 @@ pub enum XCMAppCall {
         asset_id: MainnetAssetId,
         sidechain_asset: ParachainAssetId,
         asset_kind: AssetKind,
+        min_amount: u128,
```

Comment:
Do you have special type for balance like `Balance` here? If yes, please replace it with `u128`

Reply:
the XCMApp pallet has no infrmation about Sora2 balances, the conversion is performed in substrateApp

Reply:
@N1ghtStorm you can use `MainnetBalance` type 

And I think it's better to rename this field to `minimal_xcm_amount`